lib/chef/resource/link.rb

```diff
@@ -35,62 +35,23 @@ class Resource
     class Link < Chef::Resource
       include Chef::Mixin::Securable
       resource_name :link
+      provides :link
 
-      identity_attr :target_file
-
-      state_attrs :to, :owner, :group
+      state_attrs :owner # required since it's not a property below
```

Comment:
This comes from Chef::Mixin::Securable which doesn't use properties so no state set